src/types.rs

```diff
@@ -565,6 +583,7 @@ impl SourceMap {
         SourceMap {
             file,
             tokens,
+            range_tokens: vec![],
```

Comment:
This field is only used to check if it contains particular elements. It might make more sense to use e.g. a `BTreeSet` here for faster lookups (or at least use binary search on the vector).